HW14/IhorBohush/functions_with_errors.py

```
@@ -0,0 +1,12 @@
+def greeting_by_name(name):
+    return f"Hello name!"
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)